syntax="proto2";

import "aap_protobuf/service/control/message/ByeByeReason.proto";

package aap_protobuf.service.control.message;

message ByeByeRequest
{
    required ByeByeReason reason = 1;
}
